### DISCUSSION TITLE
IR Ring Floodlight

### DISCUSSION THREAD

**Discussion by Gadgeteering** [jyjblrd/Mocap-Drones#20] [Q&A]
I don't think the **IR Ring Floodlight** is required if you fit the drone with IR LEDs.

 Has any one used [Mo Cap Markers](https://www.aliexpress.com/item/1005005045221061.html?src=google&src=google&albch=shopping&acnt=494-037-6276&slnk=&plac=&mtctp=&albbt=Google_7_shopping&albagn=888888&isSmbAutoCall=false&needSmbHouyi=false&albcp=17866112291&albag=&trgt=&crea=en1005005045221061&netw=x&device=c&albpg=&albpd=en1005005045221061&gad_source=1&gclid=CjwKCAiAnL-sBhBnEiwAJRGiggAopaC4w7qOBuyjlo7_yASx7xRosGa61sHv1pigdnzi3iSNWOJI_xoCMfcQAvD_BwE&gclsrc=aw.ds&aff_fcid=2e6a3a097edc43879a1e249e09e8dd97-1703953365906-06644-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=2e6a3a097edc43879a1e249e09e8dd97-1703953365906-06644-UneMJZVf&terminal_id=afbd6b6501ab4034b0384ef5ef39f96c&afSmartRedirect=y)

**Comment by gumby0q** [comment]
Markers should work, but with distance they are not so bright.
I have tried DIY markers with pretty strong IR ring. With original pseye lenses the result is total shit far then 1.5m. Use custom lenses.
Use LED. 
![image](https://github.com/user-attachments/assets/ceb0190f-f340-43ab-b754-34b2ca32f5d4)
SFH 4556 Osram - the best choise
also diffusor is pretty helpfull
https://www.aliexpress.com/item/1005001807749719.html
